aries-site/src/examples/components/fileinput/FileInputSimpleExample.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { Box, FileInput } from 'grommet';
+
+export const FileInputSimpleExample = () => (
+  <Box align="center" pad="large">
+    <FileInput
+      messages={{
+        dropPrompt: 'Drag and Drop',
```

Comment:
Designs seem to call for `Drag and drop or`

Reply:
Think its just Drag and drop
<img width="330" alt="Screen Shot 2021-05-05 at 8 06 35 AM" src="https://user-images.githubusercontent.com/42451602/117154138-d075e480-ad78-11eb-911b-e64e502f4dae.png">